bench/tool/tool.ml

```diff
@@ -43,22 +48,25 @@ let wait_pid =
       with Sigchld -> ()
     end;
     Sys.set_signal Sys.sigchld Signal_default;
-    let waited_pid, status = Unix.waitpid [] (-pid) in
+    let waited_pid, status, { Rusage.utime; stime; maxrss; _ } = wait4 [] pid in
```

Comment:
It should be `-pid` with a minus because we want to wait for the whole group (we use `setpgid` and everything somewhere else).

Reply:
My bad